lms/static/scripts/frontend_apps/components/test/SubmitGradeForm-test.js

```diff
@@ -189,6 +189,13 @@ describe('SubmitGradeForm', () => {
   });
 
   context('when fetching a grade', () => {
+    it('sets the defaultValue prop to an empty string if the grade is falsey', async () => {
```

Comment:
What if the grade is 0 (number)? What should happen then?

Reply:
good q. Lets handle that in another discussion since submitting 0 throws a 400 error